src/pages/editor-page/canvas/table-node.tsx

```diff
@@ -75,10 +75,12 @@ export const TableNode: React.FC<NodeProps<TableNodeType>> = ({
                 className="h-2 rounded-t-lg"
                 style={{ backgroundColor: table.color }}
             ></div>
-            <div className="flex items-center h-9 bg-slate-200 px-2 justify-between group">
+            <div className="flex items-center h-9 px-2 justify-between group">
```

Comment:
this change also changes the color of the light mode table header, how about adding something like `dark:bg-...` instead?